util/settings/settings.go

```diff
@@ -81,6 +81,7 @@ type RepoCredentials struct {
 	PasswordSecret        *apiv1.SecretKeySelector `json:"passwordSecret,omitempty"`
 	SSHPrivateKeySecret   *apiv1.SecretKeySelector `json:"sshPrivateKeySecret,omitempty"`
 	InsecureIgnoreHostKey bool                     `json:"insecureIgnoreHostKey,omitempty"`
+	Insecure              bool                     `json:"insecure,omitempty"`
```

Comment:
should this be commented?

Reply:
Yes, you are right. Will fix it with one of the next commits.